app/scripts/lib/metrics.js

```diff
@@ -484,10 +484,16 @@ define(function (require, exports, module) {
 
     logFlowEvent (eventName, viewName) {
       this.logEvent(marshallFlowEvent(eventName, viewName));
+      // We need to use server-side timestamps on flow events,
+      // so flush them to the server immediately.
+      this.flush();
```

Comment:
I guess a similar comment to what I had in the previous PR: we are already hammering the metrics endpoint. Can we do something different here, rather than flushing the metrics a lot. For example, can the flowId be generated on the server and memcached there that way the flow is setup with the proper (server) timestamp?  Any other solutions?  

Just to clarify where you say "clients are sending garbage flow" is it because the timestamps are totally wrong?